distribution/helm/vector-agent/templates/secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- with .Values.secrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "libvector.fullname" . }}
+  labels:
+    {{- include "libvector.labels" . | nindent 4 }}
+type: Opaque
+data:
+  awsAccessKeyId: {{ default "MISSING" .Values.secrets.aws.accessKeyId | b64enc | quote }}
+  awsSecretAccessKey: {{ default "MISSING" .Values.secrets.aws.secretAccessKey | b64enc | quote }}
```

Comment:
Is there any value in making the default `MISSING`? As in, the value gets base64 encoded, and I assume (/ hope) that no logging occurs that prints the actual value of these secrets, so it's not like you'll ever see it ever again?

Reply:
_Probably_ not - that pattern was copied from the datadog helm chart, empty or MISSING would end up with vector erroring trying to auth with AWS in this case

Reply:
Probably removed if/as we do arbitrary k/v's here rather than pre-defined ones 🤔